packages/dbml-core/__tests__/exporter/sqlite_exporter/input/general_schema.in.dbml

```diff
@@ -0,0 +1,57 @@
+Table "orders" {
+  "id" integer [pk, increment]
+  "user_id" integer [unique, not null]
+  "status" varchar
+  "created_at" varchar
+}
+
+Table "order_items" {
+  "order_id" integer
+  "product_id" integer
+  "quantity" integer [default: 1]
+}
```

Comment:
Although the DBML syntax is correct, this mapping table should be defined after the `products` table so that when we export to SQLite, if the foreign key constraints are added in the `CREATE TABLE "ORDER_ITEMS" ...` statement, the exported SQL is still correct.